pages/your_tasks/virtual_dev_environments.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Reproducible virtual software development environments
+description: Virtual software development environments for reproducible research 
+contributors: ["Aleksandra Nenadic"]
+page_id: virtual_environments
+related_pages: []
+---
+
+## What are virtual software development environments?
+
+A virtual software development environment helps us create an **isolated working copy** of a software project that uses a specific 
+version of a programming language interpreter/compiler (e.g. Python 3.11) together with specific versions of a number of external 
+libraries (dependencies) required by our software installed into that virtual environment. 
+
+Virtual environments are typically implemented as directories with a particular structure (usually contained within software projects 
```

Comment:
I am not sure the text (usually contained within software projects ...) adds to the explanation - consider removing or simplifying?

Reply:
Clarified via 4f3c01d